docker/Dockerfile.base

```diff
@@ -43,7 +44,7 @@ RUN apt-get update && apt-get install -y libfontconfig && rm -rf /var/lib/apt/li
 
 ENV PATH="/opt/venv/bin:$PATH"
 
-# Importing Haystack will generate and persist the json schema, we do this here for two reasons:
+# The JSON schema is lazily generated at first usage, but we do it explicitly here for two reasons:
 # - the schema will be already there when the container runs, saving the generation overhead when a container starts
 # - derived images don't need to write the schema and can run with lower user privileges
-RUN python3 -c "import haystack"
+RUN python3 -c "from haystack.utils.docker import cache_schema; cache_schema()"
```

Comment:
I checked the code, and this can result in an exception. I hope it will fail to create a docker image, right?

Reply:
Command would exit with 1 and build would fail.